bundles/action/org.openhab.action.astro/OSGI-INF/action.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2010-2015, openHAB.org and others.
+
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" deactivate="deactivate" immediate="true" name="org.openhab.action.astro.action">
+	<implementation class="org.openhab.action.astro.internal.AstroActionService" />
+
+	<service>
+		<provide interface="org.openhab.core.scriptengine.action.ActionService" />
+		<provide interface="org.osgi.service.cm.ManagedService" />
```

Comment:
since updated() isn't implemented in your Action doesn't need to implement ManagedService